otel-extension/src/test/scala/io/scalac/mesmer/instrumentation/akka/actor/ActorMailboxTest.scala

```diff
@@ -135,7 +135,7 @@ class ActorMailboxTest
     (sut, context)
   }
 
-  private def testWithProps(props: Props, amountOfMessages: Int = 10, expectedValue: Long = 5): Any = {
+  private def testWithProps(props: Props, amountOfMessages: Int = 10, expectedValue: Long = 7): Any = {
```

Comment:
We got good result because amount of `received` and `dropped` messages was the same - it's a good think to remember in the future, to always make those asymetric